### PR TITLE
[ADD] Module to hide Odoo branding

### PR DESCRIPTION
**Pull request by adru-odoo**
### Description

*Please include a summary of the change, the project task link and/or the ID.*

Link to task: [#ID](https://www.odoo.com/web#model=project.task&id=2982799)

### All Submissions:

* [ ] My commit respects the [Odoo commit guideline](https://www.odoo.com/documentation/15.0/developer/misc/other/guidelines.html#git)
* [ ] My commit message respects the [commit template](https://github.com/odoo-ps/psbe-process/wiki/Commits-message-guidelines#template)
* [ ] I have used pre-commit
* [ ] The PR contains **only** my modification and **no other external** commit

### Sh/Runbot:

* [ ] The commits pass test and the branch is green
* [ ] Unit tests have been implemented / standard ones rewritten
* [ ] The Staging is ISO-Prod and will contain only this dev

### Upgrade:

* [ ] The data affected (*if any*) by the changes has been migrated

### CODE REVIEW

File: nelsonco_hide_brand/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```
Comment (jot-odoo) [inline nelsonco_hide_brand/__init__.py:1]: You can remove the encoding line from each python file.
Missing newline

File: nelsonco_hide_brand/views/webclient_templates.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="nelsonco_hide_brand.brand_promotion" inherit_id="website_sale.brand_promotion">
+        <xpath expr="//t[@t-call='web.brand_promotion_message']" position="replace">
+	</xpath>
```
Comment (jot-odoo) [inline nelsonco_hide_brand/views/webclient_templates.xml:5]: Since there's no children, you can put the end slash in the first xpath. `<xpath ... />`

**Review by jot-odoo — CHANGES_REQUESTED** [review]

**Comment by jot-odoo** [comment]
Also, add a link to the PR to the task itself in the "Git (URLs to Code)" section

File: nelsonco_hide_brand/views/webclient_templates.xml (hunk 2 above)
Comment (adru-odoo) [reply nelsonco_hide_brand/views/webclient_templates.xml]: should be fixed, thanks!

File: nelsonco_hide_brand/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```
Comment (jot-odoo) [inline nelsonco_hide_brand/__manifest__.py:1]: Missed one

**Review by jot-odoo — CHANGES_REQUESTED** [review]

File: nelsonco_hide_brand/__manifest__.py (hunk 3 above)
Comment (adru-odoo) [reply nelsonco_hide_brand/__manifest__.py]: oops! should be gone now

**Review by jot-odoo — APPROVED** [review]

**Pull request merged**